docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,135 @@
+---
+id: running-and-editing-timelines
+title: Running and Editing Timelines
+sidebar_label: Running and Editing Timelines
+---
+
+Running a timeline to timecode
+------------------
+
+The HTP (intensity) of fixtures in the timeline is controlled by the fader level of the handle. 
+
+1.  Ensure the **timecode link** button (bottom left of the timeline view) is set to **Linked**.
+
+2.  Push up the fader of the handle where the timeline is stored.
+
+3.  Start the timecode source. 
+
+4.  The cues in the timeline will run when the timecode reaches their programmed times.
+
+- On consoles with motorised faders and on virtual faders, the faders will move as programmed.
+
+- If you are using internal timecode you can set the timecode to run automatically when you raise the fader, and/or to stop when you lower the fader. See [Timeline Options - Fader](../timelines/timeline-options.md#fader-tab).
+
+- You can set the timeline to loop (when using internal timecode), see [Timeline Options - Timecode](../timelines/timeline-options.md#timecode-tab).
+
+- If you start the timecode at a point part way through the timeline, any active triggers will fire, but previous triggers will not - this might result in fixtures starting from a different state and changing the look.
+
+### Testing a timeline
+
+You can test a timeline without running the timecode source by setting the **timecode link** button to **Unlinked**. You can then use the Play/Pause/Rewind buttons
+in the Timeline View window to run the timeline sequence. This can be useful when you don't have control of the timecode source.                  
+
+Timeline Editing
+--------------------------
+ 
+### Selecting triggers
+
+To edit a trigger, you need to select it. There are several different ways to do this.
+-   Click on a trigger block to select all the trigger points within the block. You will see selection 
```

Comment:
Click -> tap

Reply:
Or maybe "press"

Reply:
Changed to "Tap" as that is used elsewhere. "Press" is used for touch buttons.